submissions/natalia-artsiukh/html-css-popup/index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Popup</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="header_nav">
+            <div class="nav-position">
+                <a href="#" class="link">Mail</a>
+                <a href="#" class="link">Images</a>
+                <input type="checkbox" class="check-input" id="check">
+                <label for="check" class="label-position">
+                    <span class="box-popup"></span>
+                </label>
+                <div class="popup">
+                    <div class="popup-angle"></div>
+                    <div class="popup-content">
+                        <a href="#" class="popup-item">
+                            <img src="./img/calendar.png" alt="calendar" class="popup-img">
```

Comment:
Read [this article](https://htmlacademy.ru/blog/articles/alt-text) about `alt` attribute. Sorry for rusisan but it's a good article.